### PR TITLE
Carthage  remove forum polls

### PR DESCRIPTION
**Pull request by dobertRowneySr**
Addresses #2546 
The agreed solution is to remove forum polls & votes functionality entirely from the `forum` pallet

### CODE REVIEW

File: runtime-modules/forum/src/lib.rs [hunk 1]
```diff
@@ -172,48 +166,6 @@ pub trait StorageLimits {
 
     /// Maximum total of all existing categories
     type MaxCategories: Get<u64>;
-
-    /// Maximum number of poll alternatives
-    type MaxPollAlternativesNumber: Get<u64>;
```
Comment (Lezek123) [inline runtime-modules/forum/src/lib.rs:177]: Should be also removed from `!parameter_types` in `runtime-modules/forum/src/mock.rs`

File: runtime-modules/forum/src/mock.rs [hunk 2]
```diff
@@ -811,9 +771,6 @@ pub fn edit_post_text_mock(
 }
 
 /// Change current timestamp
```
Comment (Lezek123) [inline runtime-modules/forum/src/mock.rs]: Remove the danging comment

**Review by Lezek123 — COMMENTED** [review]
LGTM, just some minor leftovers to remove.
Obviously the CLI, network tests and mappings will need to be adjusted now, but I suppose @zeeshanakram3 / @ondratra will take care of this part?

**Comment by ondratra** [comment]
> LGTM, just some minor leftovers to remove. Obviously the CLI, network tests and mappings will need to be adjusted now, but I suppose @zeeshanakram3 / @ondratra will take care of this part?

I've created an issue for that https://github.com/Joystream/joystream/issues/4075 and assigned us to it.

**Review by Lezek123 — APPROVED** [review]
LGTM

**Pull request merged**